packages/ember-views/lib/views/states/has_element.js

```diff
@@ -15,8 +16,21 @@ var hasElement = Object.create(_default);
 
 assign(hasElement, {
   $(view, sel) {
-    var elem = view.element;
-    return sel ? jQuery(sel, elem) : jQuery(elem);
+    if (isEnabled('ember-views-tagless-jquery') && view.tagName === '') {
```

Comment:
This is not handled. It would be nice to tweak the plugin to simple replace the `isEnabled` with `true` or `false` and to let Babel (or uglify) do the code elimination.